js/classes/entities/midground/enemies/flying_enemy_base.js

```diff
@@ -21,7 +21,7 @@ class FlyingEnemyBase {
         this.enemy = enemy;
         this.base = new EnemyBase(enemy, pos, scaledSize, speed, health, onDeath);
 
-        this.path = path;
+        this.path = this.relativeToAbsolute(path, pos);
```

Comment:
It would probably be good to specify through JSDoc and/or param name that `path` is now relative to the enemy's position.